plugins/modules/web_infrastructure/gunicorn.py

```diff
@@ -162,10 +157,8 @@ def main():
     pid = params['pid']
 
     # use venv path if exists
-    if venv:
-        gunicorn_command = "/".join((venv, 'bin', 'gunicorn'))
-    else:
-        gunicorn_command = 'gunicorn'
+    opt_dir = [os.sep.join(venv, 'bin')] if venv else None
+    gunicorn_command = module.get_bin_path('gunicorn', opt_dirs=opt_dir)
```

Comment:
This is a breaking change if `venv` is specified: if `gunicorn` exists in /sbin, /usr/sbin, or /usr/local/sbin, then `get_bin_path` will prefer that one instead of the one in `venv`. Also if `<venv>/bin/gunicorn` doesn't exist for some reason, instead of erroring out, the module will not use any other `gunicorn` it can find in `$PATH`.

Reply:
Got it, it is fixed.